docs/source/changelog.rst

```diff
@@ -1,12 +1,12 @@
 Changelog
 =========
 
-0.2.1 / 2017-??-??
+0.3.0 / 2017-??-??
```

Comment:
I wonder if we should do an incremental release before `0.3.0` if we plan to have that be the one where we swap to the google-cloud-python libraries? Seems like that PR has stalled and I don't have time to rebase it either.

Reply:
Done